src/util/parameter_indices.cpp

```diff
@@ -0,0 +1,17 @@
+/*******************************************************************\
+
+Module: Parameter indexing
+
+Author: Chris Smowton, chris.smowton@diffblue.com
+
+\*******************************************************************/
+
+#include "parameter_indices.h"
+
+std::map<irep_idt, std::size_t> get_parameter_indices(const code_typet &fn_type)
```

Comment:
function comment block (also explain the rationale where this function might be useful)